src/Client.js

```diff
@@ -810,6 +810,45 @@ class Client extends EventEmitter {
         }
     }
 
+    /**
+     * Get the formatted number of a WhatsApp ID. 
+     * Will return null if the number is not registered on WhatsApp.
+     * @param {string} number Number or ID ("@c.us" will be automatically appended if not specified)
+     * @returns {Promise<string|null>}
+     */
+    async getFormattedId(number) {
+        if(!number.endsWith('@s.whatsapp.net')) {
+            number = number.replace('c.us', '');
+            number += 's.whatsapp.net';
+        }
+
+        try {
+            return await this.pupPage.evaluate(async numberId => {
+                return window.NumberInfo.formattedPhoneNumber(numberId);
+            }, number);
+        } catch(_) {
```

Comment:
What does this throw that needs to return null? In what cases is this triggered. Maybe we should be more explicit of which error we're catching.

Reply:
I don't quite remember, but seems like it doesn't throw an error. I'll remove it.